src/components/home/index.css

```diff
@@ -0,0 +1,34 @@
+.relatedCardsContainer {
+  background-color: #c8ccd6;
+  padding: 5rem 10% 5rem 10%;
+}
+
+.card {
+  transition: 0.5 ease;
+  cursor: pointer;
+  color: gray;
+}
+
+.card:hover {
+  transform: scale(1.1);
+  box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.6);
+}
+
+.card-img-top {
+  width: auto !important;
+  margin-bottom: 20px;
+}
+
+#scondAndThirdCardLogo {
```

Comment:
in general we never use IDs, make this a class. and fix typo, and maybe just call it cardLogo